From 600f67b76c133e3b34da768b0d13058b00610a9c Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Wed, 10 Apr 2013 15:55:35 +0200
Subject: [PATCH 1/3] configure: remove libpanelapplet2 dependency

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>

Upstream-status: Inappropriate [version no longer being developed]
---
 configure.in | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/configure.in b/configure.in
index ca6b5fe..d976856 100644
--- a/configure.in
+++ b/configure.in
@@ -116,11 +116,6 @@ PKG_CHECK_MODULES(GTK, gtk+-2.0 >= $GTK_REQUIRED)
 AC_SUBST(GTK_FLAGS)
 AC_SUBST(GTK_LIBS)
 
-dnl -- check for libpanelapplet (required) ------------------------------------
-PKG_CHECK_MODULES(GNOME_APPLETS, libpanelapplet-2.0 >= $LIBPANEL_REQUIRED)
-AC_SUBST(GNOME_APPLETS_CFLAGS)
-AC_SUBST(GNOME_APPLETS_LIBS)
-
 dnl -- check for libpanelapplet3 (required) ------------------------------------
 PKG_CHECK_MODULES(GNOME_APPLETS3, libpanelapplet-3.0 >= $LIBPANEL3_REQUIRED)
 AC_SUBST(GNOME_APPLETS3_CFLAGS)
-- 
1.8.1.4

